Stripe/NSError+STPCustomerContext.m

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
What about adding a private header (StripeError+Private) for all those extra errors, then?